packages/manager/apps/container/src/container/legacy/server-sidebar/universe/public-cloud/PublicCloudSidebar.tsx

```diff
@@ -40,7 +40,7 @@ export default function PublicCloudSidebar() {
 
   const menu = useMemo(() => {
     if (!availability) return [];
-    const menuItems = getPciProjectMenu(projectId, region, (...args) =>
+    const menuItems = getPciProjectMenu(projectId, region, (...args):string =>
```

Comment:
formatting is good here ?

Reply:
I had typescript error without

Reply:
Maybe it's in function directly you can typed https://github.com/ovh/manager/blob/fdbc37a1ae7c3e131094f5de8fe5dddf7720000c/packages/manager/apps/container/src/container/legacy/server-sidebar/universe/public-cloud/pci-menu.ts